delta-api/src/main/java/io/cdap/delta/api/assessment/TableAssessment.java

```diff
@@ -26,15 +26,21 @@
  */
 public class TableAssessment {
   private final List<ColumnAssessment> columns;
+  private final List<Problem> features;
```

Comment:
Would it be better to call this `problems` to be more intuitive?

Reply:
the mocks have a 'features' list of problems and a 'connectivity' list of problems, which is where this name comes from, though I agree it's confusing. I'll rename to featureProblems.